feat: Update release health adoption docs (#3228)
getsentry/sentry-docs: src/docs/product/releases/health/index.mdx

### Active Users

> Users that started the application at least once in the last 24 hours.

Reviewer: ```suggestion
Users who started the application at least once in the last 24 hours.
```

### Crash

> The app had an explicit unhandled error or hard crash. You'll typically be able to view the corresponding issue in [sentry.io](https://sentry.io) that captures this event, and errors that did not cause the end of the application should not be included. To search for them in **Discover** or on the **Issues** page, filter by `error.unhandled:true `. The number of unhandled events is not expected to match the number of crashed sessions because sessions are not subject to [Inbound Filters](/product/data-management-settings/filtering/) and [Sampling](/platform-redirect/?next=/configuration/sampling/).

Reviewer: ```suggestion
The app had an explicit unhandled error or hard crash. You'll typically be able to view the corresponding issue in [sentry.io](https://sentry.io) that captures this event, and errors that did not cause the end of the application should not be included. To search for them in **Discover** or on the **Issues** page, filter by `error.unhandled:true `. The number of unhandled events is not expected to match the number of crashed sessions because sessions are not subject to [Inbound Filters](/product/data-management-settings/filtering/) or [Sampling](/platform-redirect/?next=/configuration/sampling/).
```

### Crash Free User

Percentage of distinct users who did not experience a crash during the specified time period.

### Crash Free Sessions

> Percentage of sessions in the specified time range that did not end by a crash of the application.

Reviewer: ```suggestion
Percentage of sessions in the specified time range not ended by a crash of the application.
```

### Crashed Users

Number of users that experienced a crash in the specified time range.

### Release Adoption

Depending on selected Display option, we show either session or user based adoption.

User Adoption is the number of users that started at least once during the last 24 hours of a specific release.
Session Adoption is the number of sessions in the last 24 hours of a specific release.

> Adoption in the progress bar is calculated as a percentage of last 24 hour users/sessions in this release vs all releases in the project.

Author: Or maybe this?
```suggestion
Adoption in the progress bar compares the sessions or users of a release with the total sessions or users for this project in the last 24 hours.
```

> Depending on selected Display option, we show either session or user based adoption.
> 
> User Adoption is the number of users that started at least once during the last 24 hours of a specific release.
> Session Adoption is the number of sessions in the last 24 hours of a specific release.
> 
> Adoption in the progress bar is calculated as a percentage of last 24 hour users/sessions in this release vs all releases in the project.

Reviewer: ```suggestion
Depending on the selected display option, we show either:
- Session adoption: the number of sessions of a specific release in the last 24 hours.
- User based adoption: the number of users who started the application at least once during the last 24 hours of a specific release.

Adoption in the progress bar is calculated as a percentage of last 24 hour users/sessions in this release vs all releases in the project.
```
